cyclops/models/wrappers/sk_model.py

```diff
@@ -81,66 +102,142 @@ def initialize_model(self, **kwargs):
 
         return self
 
-    def find_best(
+    def find_best(  # pylint: disable=too-many-branches
```

Comment:
Note that there is a [base class](https://github.com/VectorInstitute/cyclops/blob/7f30a685e432e856b2d987b7329248e16220e456/cyclops/models/wrappers/base.py) that is intended to serve as a template for implementing model wrappers (this one didn't inherit it because the methods were added dynamically), and it has this signature for the `find_best` method:
```python
def find_best(
        self,
        X,
        y,
        parameters: Union[Dict, List[Dict]],
        metric: Optional[Union[str, Callable, Sequence, Dict]] = None,
        method: Literal["grid", "random"] = "grid",
        **kwargs,
    )
 ```
 I think the `SKModel` should follow the same signature. Otherwise, the signature for the base and PTModel wrappers should be updated.
 
 Ditto for the other methods.